main.go

```diff
@@ -6,18 +6,22 @@ import (
 	"errors"
 	"flag"
 	"fmt"
-	"log"
 	"math/rand"
 	"net"
+	"net/http"
+	_ "net/http/pprof"
```

Comment:
is the underscore expected here?

Reply:
Yes it imports with only the side-effects. I don't understand it much, myself